rootconf/default/etc/init.d/console-logger.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+#  Copyright (C) 2016 Audi Hsu <audi.hsu@quantatw.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+##
+## redirect console_log_file and tee_log_file to /dev/console
+##
+
+. /lib/onie/functions
+
+touch $console_log_file && /usr/bin/tail -f $console_log_file > /dev/console &
```

Comment:
Looking for this whole patch, it seems we can simplify it a little bit.

Do we really need both `$console_log_file` and `$tee_log_file`?

It seems we only need `$tee_log_file`.